lucet-runtime/lucet-runtime-internals/src/instance.rs

```diff
@@ -794,7 +810,7 @@ impl Instance {
 
     /// Run a function in guest context at the given entrypoint.
     fn run_func(&mut self, func: FunctionHandle, args: &[Val]) -> Result<RunResult, Error> {
-        if !(self.state.is_ready() || (self.state.is_fault() && !self.state.is_fatal())) {
+        if !(self.state.is_ready() || (self.state.is_faulted() && !self.state.is_fatal())) {
```

Comment:
this ought to be `self.is_faulted()`, no?

Reply:
The new accessors are defined in terms of the methods on `State`, so in release mode it'd probably be equivalent.

Reply:
Oh! I'd skimmed over the rename in `state.rs` and was just missing that.